app/models/host.rb

```diff
@@ -487,7 +487,7 @@ def disable_vmotion
   def vmotion_enabled?
     msg = validate_vmotion_enabled?
     if msg[:available] && respond_to?(:vim_vmotion_enabled?)
-      check_policy_prevent("request_host_vmotion_enabled", "vim_vmotion_enabled?")
+      vim_vmotion_enabled?
```

Comment:
@lfu I think removing this check makes sense, but for a slightly different reason.  This looks like it was a typo more than an intentional change in event name.  However, I do not believe we should be doing the check here for `vmotion_enabled?`.  This method should return if it is supported from the provider and we are doing the policy check if it is requested.

On a side note, these methods should be refactored to move into the VMware specific layer since they all require `vim_*` methods.  Probably along the lines of how we do the `raw_*` methods for other provider methods.